cynthia_websites_mini_client/src/cynthia_websites_mini_client/dom.ts

```
@@ -1,3 +1,5 @@
+import { Error, Ok } from "../../prelude";
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)